projects/cashmere-examples/src/lib/navbar-app-switcher/navbar-app-switcher-example.component.html

```diff
@@ -35,20 +35,45 @@
     </hc-navbar-mobile-menu>
 </hc-navbar>
 
-<hc-pop #appSwitcher><hc-app-switcher iconHeight="100"></hc-app-switcher></hc-pop>
+<hc-pop #appSwitcher [showArrow]="false" horizontalAlign="end">
+    <hc-app-switcher iconHeight="50"></hc-app-switcher>
+</hc-pop>
 
-<hc-pop #options>
-    <ul class="list-options">
```

Comment:
Will this still work?

Reply:
Yup, I didn't remove that class, just added a comment to it saying that it's deprecated.  So this will still work if someone was using that previous list style